Add email announcing SR2021 and CTA (#38)
srobo/volunteer-emails: SR2021/2020-11-02-sr2021-announcement.md

---

> to: All Volunteers

Reviewer: Presumably this is actually to the "potential volunteers" mailchimp list?  (As opposed to people who are already enlisted in the kit team and/or competition team.)

Author: Yes

Reviewer 2: Is there a problem with it going to all volunteers? 

Last competition cycle we got feedback that some volunteers not on the competition team were not kept in the loop as to what was going on. To me, this email feels fine to send to everyone and means all volunteers regardless of whether they are in a team or not will know the plan for this years competition.

Author: https://github.com/srobo/volunteer-emails/pull/38/commits/48c90b25c62578d107b24ad1dc0c133792ca7838

Reviewer 2: @RealOrangeOne Could you clarify why some volunteers wont receive this email? I might just be missing something

Author: Definitely agree that we want to cast the net out wide on this one.

"potential volunteers" is the name of the main mailing list in mailchimp. There's now a "volunteers" list, which is those with a gsuite account. The former should be a superset of the latter. 

This email is going to the "potential volunteers" mailing list, as has 99% of the other emails on this list. https://github.com/srobo/volunteer-emails/commit/48c90b25c62578d107b24ad1dc0c133792ca7838 was simply updating the frontmatter to reflect reality (and the other email templates)

Reviewer 2: Ahh ok, that makes sense. Thank you for clarifying :)

subject: Announcing Student Robotics 2021
---

Hi All,

> We're pleased to announce some details for Student Robotics 2021 - and we need your help to make it happen.

Reviewer: To reduce confusion, please can you make it clear that this is a message from the competition team (committee).  It is probably worth describing what the competition team is as well, as there are many people on this mailing list who do not know what it is.

Author: https://github.com/srobo/volunteer-emails/pull/38/commits/9dd3c17be18feece38e0d7a920a8197e78d2d408

## Competition

> Unfortunately, we are unable to run a physical competition in a safe way. Instead, we will be running a virtual competition in a simulated environment. The competition year will be very similar to what you're used to. In a few weeks, we'll kick off the competition with a kickstart, which will culminate in a final competition 1st May.

Reviewer 2: ```suggestion
Unfortunately, we are unable to find a way to run a physical competition in a COVID safe way. Instead, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment. The competition's time scale will be very similar to what you're used to, with just a few short weeks until we'll kick off the competition with our Kickstart event. Teams will then have roughly 7 months to build their robot before the final competition on 1st May.
```

Use update text from teams email https://github.com/srobo/team-emails/pull/65/commits/3b5a7ab608a3cbd0aebea6bf01b8258312612dd9

## Kickstart

> Kickstart will be streamed live on **21st November 2020**. Here we'll announce the game for this year and go into further details on how the competition year will work.

Reviewer 3: Sorry, I couldn't attend the session this evening so I'm not sure what the plan is for this. Will the event be recorded as well as streamed? If so, should this be mentioned here?

Author: No problem :smile: YouTube will automatically keep an "archive" of the stream up and available after the event, so it can be rewatched by teams and volunteers as needed.

> Kickstart will be streamed live on **21st November 2020**. Here we'll announce the game for this year and go into further details on how the competition year will work.

Reviewer: People on this mailing list do not necessarily know what "kickstart" is.

Author: https://github.com/srobo/volunteer-emails/pull/38/commits/59891ef829d7610864082b08eb6859b90e02f42b

## We need your help!

> Student Robotics is nothing without its volunteers, and we need you to help make it happen! Whilst there's no in-person kickstart, competition or other events to organise, running a virtual competition brings its own challenges, and there's still a lot of work to do.

Reviewer 3: ```suggestion
Student Robotics is nothing without its volunteers! Whilst there's no in-person Kickstart, competition, or other events to organise, running a virtual competition brings its own challenges, and there's still so much to do. We need your help to make it happen!
```

> Student Robotics is nothing without its volunteers! Whilst there's no in-person Kickstart, competition, or other events to organise, running a virtual competition brings its own challenges, and there's still so much to do. We need your help to make it happen!

Reviewer: This email is about people volunteering for the competition team, so please rephrase the above to be about the competition team rather than "Student Robotics" as a whole.

Author: It's volunteering to help make the competition events, which is separate from "volunteering for the competition team" and "recruiting for the competition team".

It mentions what'll be covered in the meeting, which I think covers the scope just fine.

Author: https://github.com/srobo/volunteer-emails/pull/38/commits/59048e8fd7c0c849995e6929761e7d1870a69ba9

If you're interested in getting involved, whether by helping with development on our [competition simulator](https://github.com/srobo/competition-simulator), organising events, communicating with teams or anything else, we'd love to hear from you.

> We're running an event on **5th November 2020** at 19:45 (UK time) to onboard people into helping out with how the simulator works, how the competition year will work, and of course answer any questions.

Reviewer 3: ```suggestion
We'll be running an onboarding session on **5th November 2020** at 19:45 (UK time) for any potential volunteers, getting them familiar with the simulator, explaining how competitions will work this year, and of course answering any questions!
```

> If you're interested in helping out, please come along! If you're interested in helping out but can't make that time, [let us know](competition-team@).

Reviewer 2: Not seen this syntax before for email links. Is it shorthand or an incomplete link?

Author: Incomplete link. Woops!
